creator/settings/development.py

```diff
@@ -68,6 +67,7 @@
     "creator",
     "corsheaders",
     'creator.ingest_runs.apps.IngestRunsConfig',
+    "creator.studies.apps.StudiesConfig",
```

Comment:
You should be able to use `creator.studies` here.

Reply:
The signal I created doesn't work using `creator.studies`

Reply:
@gsantia if you add something like this https://github.com/kids-first/kf-api-study-creator/blob/89176bd16e9fe2ea2cefdd8eb049b41b8742264e/creator/data_templates/__init__.py into your `apps/__init__.py` you can just have `creator.studies` in your settings files